airbyte-ci/connectors/live-tests/src/live_tests/commons/connector_runner.py

```diff
@@ -168,7 +168,7 @@ async def _run(
             http_dump=await self.http_proxy.retrieve_http_dump() if self.http_proxy else None,
             executed_container=executed_container,
         )
-        await execution_result.save_artifacts(self.output_dir, self.duckdb_path)
+        await execution_result.save_artifacts(self.output_dir, self.duckdb_path if airbyte_command == "read" else None)
```

Comment:
Can we add a mapping like: 
```python
persist_command_to_duck_db = {
"read": True,
"write": True,
"spec": False,
"discover": False
}
```
```suggestion
        await execution_result.save_artifacts(self.output_dir, self.duckdb_path if persist_command_to_duck_db[airbyte_command] else None)
```

With a comment saying why we disable persistance on other commands?